src/assets/js/avatars.js

```diff
@@ -0,0 +1,5 @@
+export default function avatarUrl(profilePictureUrl) {
+  const url = new URL(profilePictureUrl, window.location.href);
```

Comment:
Seems like we may want to use a more specifically constructed hostname? Maybe not, but just using the window.location.href for the host name feels like a shortcut.

Reply:
`URL` need a full url as second argument passing just a host name errors:
```js
> new URL("/foo", "people.mozilla.com");
> TypeError: people.mozilla.com is not a valid URL.
```